common/http.hpp

```diff
@@ -272,9 +272,10 @@ class OutgoingRequest : public Request {
 
 class Stream;
 
-class IncomingRequest : public Request,
-						virtual public io::Canceller,
-						public enable_shared_from_this<IncomingRequest> {
+class IncomingRequest :
+	public Request,
+	virtual public io::Canceller,
+	public enable_shared_from_this<IncomingRequest> {
```

Comment:
From the range-diff it looks like almost this entire commit stems from the `SO_KEEPALIVE` commit, where this formatting was changed. So you could perhaps get rid of it by squashing it there.

Reply:
Well, yes and no. 

I touched the file in that commit. But not this part of the file...

So the formatting has changed, since the last time we touched  the file.

I therefore judged to keep it out of the commit, as it is not related to the functionality change, but feel free to correct me still.